WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -63,13 +61,12 @@ struct CartView: View {
                             )
                     }
                     .padding(.horizontal, Constants.itemHorizontalPadding)
```

Comment:
Pre-existing, but I noticed that the checkout button isn't correctly positioned in line with the items – it shouldn't have any extra horizontal padding on the trailing edge.

### Designs
![Designs showing alignment of `Clear` button with items in cart](https://github.com/user-attachments/assets/c8e3ff6a-f2cb-437b-b90a-d94766e21105)


### Screenshot
![Screenshot showing `Clear` button inset from items in cart](https://github.com/user-attachments/assets/3368de54-f752-41ea-b78a-58eb2632cd76)

Reply:
Good catch, updated: https://github.com/woocommerce/woocommerce-ios/pull/13744/commits/9071348d967da09938c1acdd9b56a9c45cdbd44c